security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,68 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
+
+* For prior versions of Compliance Operator, the log rotation function depended on finding the logrotate file in the /etc/cron.daily folder. With this release, Compliance Operator works with the `logrotate.timer` service. This provides reliable log rotation behavior from Compliance Operator. 
+// For more information, see (link:https://issues.redhat.com/browse/CMP-3172[CMP-3172]).
+
+* For previous versions of Compliance Operator, it is possible for the `STIG ID` to be omitted from the compliance report. These omissions were caused by missing `stigref` and `stigid` values. With this release, the omissions have been corrected and now `STIG ID` reliably shows up in the compliance report. 
+// For more information, see (link:https://issues.redhat.com/browse/OCPBUGS-60143[OCPBUGS-60143].
+
+* Prior to this release, Compliance Operator STIG control CNTR-OS-000720 selected rule 'rhcos4-audit-rules-suid-privilege-function', but since the rule was not available in Compliance Operator, no output was generated. With this release, rule 'rhcos4-audit-rules-suid-privilege-function' is now available in Compliance operator and listed in the scan output. For more information, see (link:https://issues.redhat.com/browse/CMP-3558[CMP-3558]).
+
+* In previous versions of Compliance Operator, scanning with the `ocp4-stig` profile would fail for the rule `ocp4-stig-modified-audit-log-forwarding-uses-tls` even if TLS is enabled correctly. This would occur because the `tls://` field is no longer required by the `ClusterLogForwarder` resource, causing the scan output to show an incorrect `FAIL` result. With this release, the protocol prefix is not required and the scan output produces correct results. For more information, see (link:https://access.redhat.com/solutions/7016445[routes-protected-by-tls compliance check failing when ODF 4.11 is installed]).
+
+* Previously, there was no automated method to check if API servers were using unsupported configuration overrides as recommended by CIS Benchmark control 1.2.31 or 1.2.33. This release provides dedicated rules for checking for unsupported configuration overrides. 
+* For prior releases of Compliance Operator, `resource-requests-limits` rules were missing a variable reference and produced output on that basis. With this release, the variable reference is available to `resource-requests-limits` rules and the erroneous output is eliminated. For more information, see (link:https://issues.redhat.com/browse/CMP-3582[CMP-3582]).
```

Comment:
Add an empty line before this line.  
Better to update as below as the fix works for all rules.
For prior releases of Compliance Operator,  many rules missing a variable reference in the annotation, such as rule `resource-requests-limits`. With this release, the variable reference is available for rules and the erroneous output is eliminated.